src/exporter/trace/stdout/mod.rs

```diff
@@ -0,0 +1,49 @@
+use crate::exporter::trace;
+use std::any;
+use std::sync::Arc;
+
+#[derive(Debug)]
+/// Exporter
```

Comment:
Rust conveniently adds doc scaffolding out-of-the-box for all public code, so doc comments specifying the names of structs (and functions, etc.) are unnecessary.

Reply:
https://github.com/open-telemetry/opentelemetry-rust/blob/master/src/lib.rs#L205 - This does not allow me to pass the lint without the comments.